### PR TITLE
Add Department for International Trade organisation

### PR DESCRIPTION
**Pull request by nickcolley**
Updates whitehall to the newest `govuk_frontend_toolkit` version.

https://github.com/alphagov/govuk_frontend_toolkit/compare/v4.8.1...v4.16.0

I've made sure to test the places where the newer version introduces potentially breaking changes (adds `display: block` to the heading mixins)

Removes the organisation colours since that is now in `govuk_frontend_toolkit`

Before

screen shot 2016-08-16 at 12 37 07

After

screen shot 2016-08-16 at 12 36 58

We'll need to swap to the new brand color and crest in the admin once this ships.

As work part of
https://trello.com/c/4a1yrsxj/468-add-dit-crest-logo-same-as-bis-crest-small
https://trello.com/c/LL6a7onN/464-update-dit-colour-label-small

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,4 @@
 .yardoc/
 /doc/
 /failures
+.DS_Store
```
Comment (binaryberry) [inline .gitignore]: This should be removed from the commit

Comment (nickcolley) [inline .gitignore]: Is there a reason why? I don't want to accidentally commit those files.

Comment (binaryberry) [inline .gitignore]: Aorry, I didn't see it was the gitignore. My bad!

Comment (cbaines) [inline .gitignore]: Normally for platform specific files, just include that in your `core.excludesFile`, which is described at the lower part of the description here https://git-scm.com/docs/gitignore

Comment (boffbowsh) [inline .gitignore]: What Chris said. `.DS_Store` should be treated as an environment-specific file in the same vain as a `*.swp` Vim swapfile or a `sublime-project`.

Comment (nickcolley) [inline .gitignore]: I'd prefer to make repo's easier to contribute to for new contributors since this is just boilerplate that can be part of all repos (like most open source projects do).

All dependencies should be self contained with locally with each repo, for example we wouldn't expect people to run their own linter.

That said I imagine this is one of those 'tabs vs spaces' things so will change if needed.

tl;dr contributors > conventions

Comment (boffbowsh) [inline .gitignore]: Good point, and we're bike shedding at this point anyway. It does no harm to have this file ignored, and almost all of our developers are on Macs. Happy for this to remain 👍

File: app/assets/stylesheets/frontend/styleguide/_colours.scss [hunk 2]
```diff
@@ -19,34 +19,6 @@ $inside-gov-nav-dropdown: #888;
 $coalition-green: #85994b;
 $g8-brand: #c10019;
 
-// Web safe department colours
```
Comment (boffbowsh) [inline app/assets/stylesheets/frontend/styleguide/_colours.scss]: Can you explain in this commit message please why these are removed, for future-us?

**Comment by boffbowsh** [comment]
Small commit message related comment, otherwise :+

**Comment by nickcolley** [comment]
Great point, updated the commit with the info 👍

**Comment by boffbowsh** [comment]
💥 👍
